src/services/blocks/BlocksTraceService.ts

```diff
@@ -0,0 +1,102 @@
+import { BlockHash } from '@polkadot/types/interfaces';
+import { InternalServerError } from 'http-errors';
+
+import {
+	BlocksTrace,
+	BlocksTraceOperations,
+} from '../../types/responses/BlocksTrace';
+import { AbstractService } from '../AbstractService';
+import { BlockTraceResponse, isBlockTrace, isTraceError, Trace } from './trace';
+
+const DEFAULT_TARGETS = 'pallet,frame,state';
+// :extrinsic_index & frame_system::Account
+const DEFAULT_KEYS =
```

Comment:
what does `DEFAULT_KEYS` mean?

Reply:
LMK what you think about the new comment.

Reply:
It's great makes it more clear to me at least